onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -176,6 +183,11 @@ def quantize_static(
 
     quantizer.quantize_model()
     quantizer.model.save_model_to_file(model_output, use_external_data_format)
+    if not pre_processed:
```

Comment:
why add the logic twice?

Reply:
could you please add same logic in quantize_dynamic?

Reply:
Too many logging info during calibration. Print the warning twice to increase its visibility.

About dynamic quantization, can we address that in another PR?